app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
Import Ordering: 
It's a good practice to order imports in a certain way. Typically, you'd start with standard library imports, then third-party library imports, and finally your project's local imports.